src/pages/reports.en.tsx

```diff
@@ -40,10 +40,8 @@ const ReportCard: React.FC<ReportCardInfo> = (props) => (
     <div className="jf-report-card columns is-paddingless is-multiline has-background-white">
       <div className="column is-6 is-paddingless">
         <Img
-          fluid={props.image.fluid}
+          fluid={{ ...props.image.fluid, aspectRatio: 1 }}
```

Comment:
(for clarification) does this only apply to desktop view? not sure i understand what in the code is making the desktop image a square and the mobile one a rectangle

Reply:
You're right, this doesn't make sense. I was confused about what that option was doing - I think it's actually working fine before and I shouldn't have messed with it. I had previously been struggling to get the image to maintain the aspect ratio and come in and out of view as the page size changed, and thought this was a better way to do that, but it's not doing what I thought.